### ISSUE TITLE
Include vector data

### ISSUE THREAD

**Issue by mapsam** [mapbox/gdal-tiling-bench#6]
Right now `gdal-tiling-bench` is only testing agains TIF files but with the advancement of vector data inputs in mapnik, it seems important to benchmark on file types such as `GeoJSON`  and `Shapefiles`.

Additionally running benchmarks against different instances of these vector data will be helpful. This includes points, lines, polygons, multipolygons, etc. This will allow us to begin bechmarking against specific operations running in GDAL/mapnik to get a more defined view into where bottlenecks.

I like the idea of keeping these large files up on s3, so will continue working that way unless we are using _tiny_ files.

Any input from @flippmoke @jakepruitt on what will make this benchmarking more useful for the v2 push is appreciated!
### Todo
- [x] GeoJSON test cases including different data types and sizes
- [ ] Shapefile additions with different data types and sizes (@springmeyer mentioned working with indexed tiles here?)

**Comment by sgillies** [comment]
@mapsam GeoJSON fixtures with differently/randomly sorted keys (maybe even duplicated keys!), indentation and other odd whitespace phenomena would be a good check.

**Comment by mapsam** [comment]
Good call, @sgillies - maybe some jumbo OSM extracts? I _bet_ there's some weird tags to handle there :smile:

**Comment by mapsam** [comment]
Added as geojson vector data all downloadable form s3:
- `lines` from OSM extracts, `22.2MB`, [aws url](https://s3.amazonaws.com/mapbox/playground/mapsam/addis-abeba_ethiopia_osm_line.geojson)
- `polygons` from OSM extracts, `16.3MB`, [aws url](https://s3.amazonaws.com/mapbox/playground/mapsam/bali_indonesia_landusages.geojson)
- `polygons` counties from US Census, `2.7MB`, [aws url](https://s3.amazonaws.com/mapbox/playground/mapsam/cb_2014_us_county_20m.geojson)
- `points` from OSM extracts, `13.5MB`, [aws url](https://s3.amazonaws.com/mapbox/playground/mapsam/kathmandu_nepal_osm_point.geojson)
